uaclient/cli/cve.py

```diff
@@ -0,0 +1,172 @@
+
+            if binary_pkg_info:
+                binary_pkg_cve_info = None
+                for cve in binary_pkg_info.cves:
+                    if cve.name == cve_name:
+                        binary_pkg_cve_info = cve
+                        break
+
+                if binary_pkg_cve_info:
+                    rows.append(
+                        AffectedPackage(
+                            name=binary_pkg,
+                            fix_status=binary_pkg_cve_info.fix_status,
+                            fix_origin=binary_pkg_cve_info.fix_origin,
+                            fix_version=binary_pkg_cve_info.fix_version,
+                        )
+                    )
+
+        return rows
+
+
+def _format_affected_pkgs(affected_pkgs):
+    formatted_rows = []
+
+    for affected_pkg in affected_pkgs:
+        if affected_pkg.fix_status == "fixed":
+            formatted_rows.append(
+                [
+                    "<SPACE>{}:".format(affected_pkg.name),
+                    affected_pkg.fix_status,
+                    "({})".format(affected_pkg.fix_origin),
+                    affected_pkg.fix_version,
+                ]
+            )
+        else:
+            formatted_rows.append(
+                [
+                    "<SPACE>{}:".format(affected_pkg.name),
+                    affected_pkg.fix_status,
+                    "",
+                    "",
+                ]
+            )
+
+    return formatted_rows
+
+
+def action_cve(args, *, cfg, **kwargs):
+    cve_vulnerabilities = _get_cve_vulnerabilities(args, cfg=cfg)
+
+    if args.cve not in cve_vulnerabilities.cves:
+        print(messages.CLI_CVE_NOT_AFFECTED.format(issue=args.cve))
+        return
+
+    cve_info = cve_vulnerabilities.cves[args.cve]
+    affected_pkgs_rows = _format_affected_pkgs(
+        _get_affected_pkgs(cve_vulnerabilities, cve_info, args.cve)
+    )
+    affected_pkgs_table = Table(rows=affected_pkgs_rows)
+
+    print("name:               {}".format(args.cve))
+    print(
+        "public-url:         https://ubuntu.com/security/{}".format(args.cve)
+    )
+    print(
+        "published-at:       {}".format(
+            cve_info.published_at.strftime("%Y-%m-%d")
+        )
+    )
+    print(
+        "cve-cache-date:     {}".format(
+            cve_vulnerabilities.vulnerability_data_published_at.strftime(
+                "%Y-%m-%d"
+            )
+        )
+    )
+    print(
+        "apt-cache-date:     {}".format(
+            cve_vulnerabilities.apt_updated_at.strftime("%Y-%m-%d")
+        )
+    )
+    print(
+        "priority:           {}".format(
+            cli_util.colorize_priority(cve_info.priority)
+        )
+    )
+    print("cvss-score:         {}".format(cve_info.cvss_score))
+    print("cvss-severity:      {}".format(cve_info.cvss_severity))
+    print(
+        "description: |\n  {}".format(
+            cve_info.description.replace("\n", "\n  ")
+        )
+    )
```

Comment:
Generally, I think this function will be easier to read if each new line got its own `print` call. So for this piece, it might look more like:
```suggestion
    print("description: |")
    for line in cve_info.description.split("\n"):
        print("  {}".format(line))
```
Similar refactors could be applied to `notes`, `affected_packages`, and `related_usns`

Separately, is it safe to rely on the newlines already in the description? Or should we rewrap it ourselves to max 80 characters?

Reply:
Yeah, let's be conservative and wrap that to 80 characters